src/components/BaseMap/BaseMap.module.scss

```
@@ -11,6 +11,23 @@
   z-index: theme.$zBaseMap;
 }
 
+// Reposition and restyle MapLibre's built-in scale bar to match the mockup style.
+:global(.maplibregl-ctrl-bottom-right .maplibregl-ctrl-scale) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)